python_modules/dagster/dagster/core/definitions/environment_configs.py

```diff
@@ -323,10 +348,20 @@ def construct_environment_config(config_value):
         execution=ExecutionConfig(**config_value['execution']),
         expectations=ExpectationsConfig(**config_value['expectations']),
         context=construct_context_config(config_value['context']),
+        storage=construct_storage_config(config_value.get('storage')),
         original_config_dict=config_value,
     )
 
 
+def construct_storage_config(config_value):
+    check.opt_dict_param(config_value, 'config_value', key_type=str)
+    if config_value:
+        storage_key, storage_value = single_item(config_value)
+        return StorageConfig(storage_key, storage_value)
```

Comment:
What is this key/value meant to do?

Reply:
these are poorly named. storage_mode and storage_config are more accurate